internal_task_pollers.go

```diff
@@ -29,6 +29,7 @@ import (
 	"time"
 
 	"github.com/uber-go/tally"
+	common2 "github.com/uber/cadence/common"
```

Comment:
rename to more meaningful: cadencecommon or ccommon

Reply:
Removed since not used